var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -92,12 +94,26 @@ def cmake_args(self):
             'compiler-rt'
         ]
 
+        if self.spec.satisfies('@4.2.0:'):
+            llvm_projects.append('libcxx')
+            llvm_projects.append('libcxxabi')
+
+            args = [
+                self.define('LIBCXX_ENABLE_SHARED', 'OFF'),
```

Comment:
Why static?

Reply:
this is what i have received from the team - This is needed to support the amdclang tool.  amdclang depends on
 the C++17 filesystem library, which is not yet widely supported by
 all the distros that ROCm currently supports.  The workaround is static
 link LLVM's libc++ into amdclang.